src/main/java/de/zalando/aruha/nakadi/util/UUIDGenerator.java

```diff
@@ -0,0 +1,9 @@
+package de.zalando.aruha.nakadi.util;
+
+import java.util.UUID;
+
+public class UUIDGenerator {
+    public UUID randomUUID() {
+        return UUID.randomUUID();
+    }
```

Comment:
What is the purpose of special class?
We have several places where we generate UUIDs with UUID.randomUUID(),
Now we have places with this generator and UUID call directly